AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java

```diff
@@ -1863,6 +1872,7 @@ protected Boolean task(Collection col, ProgressSenderAndCancelListener<Void> col
                 col.getModels().moveField(model, field, index);
                 col.save();
             } catch (ConfirmModSchemaException e) {
+                Timber.w(e);
```

Comment:
```suggestion
                e.log();
```

Did these two get added in the delay between you doing this and me reviewing it or did something else happen? Either way it seems like there are 2 more, but this is SO CLOSE :-)

Reply:
Normally, I added nothing. Just made the change you asked. Not even a merge conflict